src/XComponent.Functions/Controllers/RessourceController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Swashbuckle.Swagger.Annotations;
+using XComponent.Functions.Core;
+using XComponent.Functions.Core.Exceptions;
+
+namespace XComponent.Functions.Controllers
+{
+    public class RessourceController: ApiController
+    {
+        [SwaggerResponse(HttpStatusCode.OK, "Available ressource", typeof(FunctionParameter))]
+        [SwaggerResponse(HttpStatusCode.NoContent, "No ressource available")]
```

Comment:
it's a list, return an empty list if there's nothing.
it will make the service simpler

Reply:
Update documentation

Reply:
So?